docs/dev-plan/20-sales/0487-add-missing-activityservice-integration-tests.md

```
@@ -0,0 +1,258 @@
+# Sales · Add missing ActivityService integration tests
+
+| 元数据 | 值 |
+|---|---|
+| Issue | #487 |
+| 分类 | [20-sales](../README.md#12-分类总览) |
+| 优先级 | 推荐 |
+| 工作量 | 0.5-1 工作日 |
+| 依赖 | [板 #486](), [板 #452]() |
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)